src/tfvc/tfvc-extension.ts

```diff
@@ -110,14 +110,16 @@ export class TfvcExtension  {
                     const right: Uri = TfvcSCMProvider.GetRightResource(resource);
                     const title: string = resource.GetTitle();
 
+                    if (!right) {
```

Comment:
Logically, it doesn't.  It just keeps us aligned with the similar changes in the Git extension.